### DISCUSSION TITLE
Problems with egs_view

### DISCUSSION THREAD

**Discussion by zw-mphys** [nrc-cnrc/EGSnrc#1172] [General]
Hello EGSnrc users! 

I have been trying to view egs_chamber simulation geometry with egs_view. I am able select a file by running "open -a egs_view" but the application is crashing when a file is selected. I am running EGS on macOS 14.5 with arm64 architecture. I was wondering if anyone has had similar issues and knows of a resolution.

Thanks

**Comment by mchamberland** [comment]
Can you post the actual error message? Or does it just crash with no error that you can see?

Compiling egs_view is basically the final boss of installing EGSnrc on macOS.

If you have not done so, can you try following these instructions and adjusting the executable names and paths as needed for your system?

[https://github.com/nrc-cnrc/EGSnrc/issues/901#issuecomment-1196806970](https://github.com/nrc-cnrc/EGSnrc/issues/901#issuecomment-1196806970)

The link does not work properly, so use this URL and remove the spaces in it: https://github.com/nrc-cnrc/EGSnrc/iss ues/901 #issuecomment-1196806970

**Reply by zw-mphys** [reply]
Thanks for the reply! It seems that I do not get an error message and that egs_view just crashes. 

I cloned EGSnrc fresh and redid the process. It appears when I attempt to compile egs_view, I am getting the error:

 `ld: symbol(s) not found for architecture arm64
collect2: error: ld returned 1 exit status
make[1]: *** [../../bin/osx/egs_view.app/Contents/MacOS/egs_view] Error 1
make: *** [all] Error 2`

This is the same error from [(https://github.com/nrc-cnrc/EGSnrc/issues/901#issuecomment-1200336757)](https://github.com/nrc-cnrc/EGSnrc/issues/901#issuecomment-1200336757)

**Reply by mchamberland** [reply]
Yeah, I think you'll need to follow the instructions I linked to above. Use the url and remove the space in it. There are a number of modifications you need to do before compiling egs_view.

**Reply by mchamberland** [reply]
Gah I can't get any of those links to work. I'll try to post the steps later tonight.
